RSSReader/View/FeedViewController.m

```diff
@@ -78,11 +92,29 @@ - (UITableView *)tableView {
     return _tableView;
 }
 
+- (NSMutableArray *)flagsArray {
+    if (!_flagsArray) {
+        _flagsArray = [NSMutableArray new];
+    }
+    return _flagsArray;
+}
+
+#pragma mark TableViewDataSourse implementation
+
+- (void)fillFlagsArray {
+    [self.flagsArray removeAllObjects];
+    for (int i = 0; i < self.feedItemArray.count; i++) {
+        [self.flagsArray addObject:@0];
+    }
+}
+
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     FeedCell *cell = [tableView dequeueReusableCellWithIdentifier:@"cellId"];
```

Comment:
Please move NSString literal to global constant.

Reply:
fixed